aegis.sh

```diff
@@ -3,6 +3,7 @@ version: "%(tag_basename)s"
 tag: v1.4
 requires:
   - ROOT
+  - VMC
```

Comment:
This dependence is surprising; I looked in the code and the only usage of VMC is to access mag field in GeneratorCosmics:
 ```c++
 //
  auto fld = TGeoGlobalMagField::Instance()->GetField();
  if (TVirtualMC::GetMC() && TVirtualMC::GetMC()->GetMagField()) {
    fld = TVirtualMC::GetMC()->GetMagField();
  }
```
Does it make sense? I thought, that we always define field via TGeo and then set it to VMC.

Reply:
mark as resolved for now, coming back to it if necessary